src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -72,6 +72,8 @@ def load_arguments(self, _):
         c.argument('node_zones', zones_type, options_list='--node-zones', help='(PREVIEW) Space-separated list of availability zones where agent nodes will be placed.')
         c.argument('enable_pod_security_policy', action='store_true')
         c.argument('node_resource_group')
+        c.argument('enable_acr')
```

Comment:
you can declare preview here, instead of calling out in the help, check out:
   https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#preview-commands-and-arguments

Reply:
thanks, fixed in the suggested ways